### PR TITLE
Update rubygem-foreman_discovery to 19.0.1 (CP 3.1)

### PR DESCRIPTION
**Pull request by lzap**
(cherry picked from commit e4801ebee8bc32a2236bdca2166b380432065cff)

DO NOT MERGE until I test that on RC1.

### CODE REVIEW

File: packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    18.0.4
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec:11]: you somehow managed to not to drop the old 18.0.4 tarball from git, which confuses the build system

**Comment by lzap** [comment]
Allright, reversed that commit in 19.0.1, updated.

**Comment by lzap** [comment]
Let me finish testing of 19.0.1 with 3.1, then I will bump nightly to 20.0 so versioning is not confusing.

**Comment by lzap** [comment]
So it works on 3.1 RC1. Here is the deb counterpart:

https://github.com/theforeman/foreman-packaging/pull/7340

RTM TY

**Review by evgeni — APPROVED** [review]

**Pull request merged**